diff --git a/lkp-exec/install b/lkp-exec/install
index 9d77c9c07..ed7e28c7e 100755
--- a/lkp-exec/install
+++ b/lkp-exec/install
@@ -205,14 +205,6 @@ install_base_support()
 {
 	echo "Installing ruby gems..."
 
-	if [ -n "$CHINA" ]; then
-		setup_gem_china
-	fi
-	(
-		cd "$LKP_SRC"  || exit
-		bundle_install || exit
-	) || exit
-
 	create_lkp_user
 	create_lkp_dirs
 }
@@ -234,7 +226,6 @@ create_host_config
 if [[ -z "$1" ]]; then
 	install_base_support
 	all_scripts='lkp-tests makepkg'
-else
 	make_wakeup
 	all_scripts=
 fi
diff --git a/sbin/install-dependencies.sh b/sbin/install-dependencies.sh
index 5e41554bd..b41ba4f94 100755
--- a/sbin/install-dependencies.sh
+++ b/sbin/install-dependencies.sh
@@ -1,10 +1,6 @@
 #!/usr/bin/env bash
 
 INSTALL_SUCCESS_FILE=/tmp/.lkp-install-dependencies-success
-test -e $INSTALL_SUCCESS_FILE && {
-	echo "$0: already installed: $INSTALL_SUCCESS_FILE"
-	exit 0
-}
 
 SCRIPT_DIR=$(cd $(dirname $0); pwd -P)
 export LKP_SRC=$(dirname $SCRIPT_DIR)
